food-truck-api/src/main/java/food/truck/api/reviews_and_subscriptions/SubscriptionRepository.java

```diff
@@ -8,7 +8,7 @@
 import java.util.List;
 
 @Repository
-public interface SubscriptionRespository extends JpaRepository<Subscription, Long> {
+public interface SubscriptionRepository extends JpaRepository<Subscription, Long> {
```

Comment:
It took me a long time to find the difference.

Reply:
Probably took me longer. :P